app/modules/asset_groups/models.py

```diff
@@ -483,8 +485,107 @@ def anonymous(self):
 
         return self.owner is User.get_public_user()
 
+    @property
+    def mime_type_whitelist(self):
+        if getattr(self, '_mime_type_whitelist', None) is None:
+            asset_mime_type_whitelist = current_app.config.get(
+                'ASSET_MIME_TYPE_WHITELIST', []
+            )
+            asset_mime_type_whitelist = sorted(list(map(str, asset_mime_type_whitelist)))
+
+            self._mime_type_whitelist = set(asset_mime_type_whitelist)
+        return self._mime_type_whitelist
+
+    @property
+    def mime_type_whitelist_guid(self):
+        if getattr(self, '_mime_type_whitelist_guid', None) is None:
+            self._mime_type_whitelist_guid = ut.hashable_to_uuid(
+                sorted(list(self.mime_type_whitelist))
+            )
+            # Write mime.whitelist.<mime-type-whitelist-guid>.json
+            mime_type_whitelist_mapping_filepath = os.path.join(
+                current_app.config.get('PROJECT_DATABASE_PATH'),
+                'mime.whitelist.%s.json' % (self._mime_type_whitelist_guid,),
+            )
+            if not os.path.exists(mime_type_whitelist_mapping_filepath):
+                log.info(
+                    'Creating new MIME whitelist manifest: %r'
+                    % (mime_type_whitelist_mapping_filepath,)
+                )
+                with open(mime_type_whitelist_mapping_filepath, 'w') as mime_type_file:
+                    mime_type_whitelist_dict = {
+                        str(self._mime_type_whitelist_guid): sorted(
+                            list(self.mime_type_whitelist)
+                        ),
+                    }
+                    mime_type_file.write(json.dumps(mime_type_whitelist_dict))
+        return self._mime_type_whitelist_guid
+
+    def _ensure_repository_files(self, project):
+        group_path = self.get_absolute_path()
+
+        # AssetGroup Repo Structure:
+        #     _db/assetGroup/<asset_group GUID>/
+        #         - .git/
+        #         - _asset_group/
+        #         - - <user's uploaded data>
+        #         - _assets/
+        #         - - <symlinks into _asset_group/ folder> with name <asset GUID >.ext --> ../_asset_group/path/to/asset/original_name.ext
+        #         - metadata.json
+
+        if not os.path.exists(group_path):
+            # Initialize local repo
+            log.info('Creating asset_groups structure: %r' % (group_path,))
+            os.mkdir(group_path)
+
+        # Create the repo
+        git_path = os.path.join(group_path, '.git')
+        if not os.path.exists(git_path):
+            repo = git.Repo.init(group_path)
+            assert len(repo.remotes) == 0
+            gitlab_remote_public_name = current_app.config.get('GITLAB_PUBLIC_NAME', None)
```

Comment:
really these should now be git_ not gitlab_

Reply:
Yeah I think so too.  These are just the user and email address for the git commits, unrelated to gitlab.